app/templates/components/page-editor.hbs

```diff
@@ -0,0 +1,21 @@
+<div id="pageButtons">
+  <span class="flexSpace">
+    {{#paper-button data-test-clear-button raised=true onClick=(action 'clearText')}}{{t 'clear.text'}}{{/paper-button}}
+    {{#paper-button data-test-highlight-button raised=true onClick=(action 'highlight')}}{{t 'highlight.text'}}{{/paper-button}}
+  </span>
+</div>
+<div id="pageInputs">
+  <span class="flexSpace">
+    {{paper-input data-test-family-input label=(t 'current.font') value=family onChange=(action "mutFamily") icon="font_download" id="fontFamily"}}
+    {{paper-input data-test-size-input label=(t 'font.size') value=size onChange=(action "mutSize") icon="format_size" id="fontSize"}}
+    {{paper-input data-test-highlight-input label=(t 'highlighted.text') value=highlightText onChange=(action (mut highlightText)) icon="highlight"}}
+  </span>
+</div>
+
+<div class="layout-row layout-align-center-start container">
+  <img src="/bookImage.png" width="80%">
+  <span contenteditable="true" id="textArea" style={{textAreaStyle}} data-test-text-area>{{page.content}}</span>
+</div>
+<span class="layout-row layout-align-space-between-center">
+  <span style="width: 118px"></span>
```

Comment:
It is not an issue now, but keep in mind that inline styling is usually forbidden and a bad pattern.